src/index.js

```diff
@@ -32,12 +32,12 @@ import config from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
 import InstructorHome from './components/pages/InstructorHome';
 import ParentFamilyHome from './components/pages/ParentFamily/ParentFamilyHome';
-import AdminHome from './components/pages/AdminHome';
+import AdminHome from './components/pages/AdminHome/index';
 import AdminAddCourses from './components/pages/AdminAddCourse';
 import AdminCourses from './components/pages/AdminHome/AdminCourses';
 import ParentBooking from './components/pages/ParentBooking';
 import StudentHome from './components/pages/StudentHome/index';
-import Footer from './components/common/Footer';
+import Footer from './components/common/Footer/Index';
 import InstructorApplyConfirm from './components/pages/InstructorBooking/InstructorApplyConfirm';
```

Comment:
I'm not sure either of these changes should have been done. The Footer/Index one will create issues if this file is merged.